api/api.go

```diff
@@ -1,2 +1,7 @@
 //go:generate go run github.com/deepmap/oapi-codegen/v2/cmd/oapi-codegen --config=codegen.yaml ./openapi.yaml
 package api
+
+const (
+	// DefaultCreditQueryLimit specifies how many entries to return by default for credit related queries
+	DefaultCreditsQueryLimit = 1000
```

Comment:
Where is this constant used? If not in this package, it should be moved where it’s used

Reply:
It's used in the router layer, moved it there.